app/views/experiments/_experiment_result.html.haml

```diff
@@ -1,8 +1,5 @@
 %h3.subheader= "Experiment result"
-
-- unless @experiment.results.nil?
-  %p
-    #{@experiment.results}
-- unless @experiment.error_reason.nil?
-  %p
-    #{@experiment.error_reason}
+%p#results
+  //#{@experiment.results}
+%p#error_reason
```

Comment:
Jeszcze trzeba dorobić jakiś nagłówek mówiący o tym, że jest to error_reason jeśli jest dostępny - czyli podobnie jak w okienku SimulationManagera